test/fixture/fixture.test.js

```diff
@@ -6,10 +6,11 @@
+    export default JSON.parse('{}', reviver);
```

Comment:
investigate why this didn't rebuild properly

then change `build.js` to write JSON files, not JS. Make fixture.test.js static and import dynamic JSON, and import the `mod` function